linters/src/entrypoint.sh

```diff
@@ -29,6 +29,11 @@ if [[ "$BASE_COMMIT" == null ]]; then  # If this is not a pull request action it
   BASE_COMMIT=$(jq --raw-output .check_suite.pull_requests[0].base.sha "$GITHUB_EVENT_PATH")
 fi
 
+# Our Github actions are hosted in a separate repo from the ones that the actions run.
+# Git introduced a security measure for that and since we're the owner of both repos, we decided to disable it.
+# See https://impel.atlassian.net/browse/PID-51 for more.”
+git config --global --add safe.directory /github/workspace
```

Comment:
Looks fine to me, just curious about how you arrived at the setting of `/github/workspace`. I could only find reference "*" in the blog posts.

Reply:
That was part of the error message when I ran the `git status` command:

![image](https://user-images.githubusercontent.com/42940081/219420039-6d49bf0d-6996-4dc2-8ce1-b7c4ee34e158.png)

It's on the PID-51 ticket too